### PR TITLE
gds-product-introduction updates

### PR DESCRIPTION
**Pull request by martinohanlon**
Targeted updates to the course to improve issues relating to the native and cypher projection lessons and challenges.

### CODE REVIEW

**Comment by FlorentinD** [comment]
@martinohanlon thanks for updating to the course to the new cypher projection!
I think we should rework the framing of cypher vs native projection. Its not clear that native is always faster than cypher (see https://neo4j.com/docs/graph-data-science/current/management-ops/graph-creation/).

Also gave it an attempt in the linked PR.

**Comment by martinohanlon** [comment]
Thanks @FlorentinD - really helpful to get contributions.

We have plans to make some more significant changes to our catalogue of GDS courses. These changes are to address some urgent issues I noticed while doing a review. 

I'll look through the PR and push any simple changes.

**Comment by martinohanlon** [comment]
@FlorentinD I have incorporated the majority of your suggested changes in #194 and #196.

File: asciidoc/courses/gds-product-introduction/modules/2-graph-management/lessons/3-c-native-projection/lesson.adoc [hunk 1]
```diff
@@ -12,6 +13,6 @@ include::./questions/verify.adoc[leveloffset=+1]
 [.summary]
 == Lesson Summary
 
-In this lesson we went over native projections, the primary graph projection mechanism in GDS.  Native Projections are optimized for efficiency and performance to support graph data science at scale.
+In this lesson we went over native projections, the easiest graph projection mechanism in GDS.
```
Comment (adam-cowley) [inline asciidoc/courses/gds-product-introduction/modules/2-graph-management/lessons/3-c-native-projection/lesson.adoc:16]: We should avoid describing this as "easy" (as per Stefano's workshop)

File: asciidoc/courses/gds-product-introduction/modules/2-graph-management/lessons/3-c-native-projection/sandbox.cypher [hunk 2]
```diff
@@ -1,13 +1,8 @@
...
-  ],
+  [],
   // Relationship types to include in the projection
-  {
-    /* ... */
-  }
-);
-
+  []
+)
```
Comment (adam-cowley) [inline asciidoc/courses/gds-product-introduction/modules/2-graph-management/lessons/3-c-native-projection/sandbox.cypher:8]: I like the comments as placeholders here - or replicate the `????` from #196 for consistency

File: asciidoc/courses/gds-product-introduction/modules/2-graph-management/lessons/4-cypher-projection/lesson.adoc [hunk 3]
```diff
@@ -8,25 +10,39 @@
 [.transcript]
 == Introduction
 
-While the native projection is scalable and fast, its filtering and aggregation capabilities aren’t as flexible as Cypher. The Cypher projection, as its name implies, uses Cypher to define the projection pattern, and as such, enables more flexibility.
+While the native projection is easier, its filtering and aggregation capabilities aren’t as flexible as Cypher. The Cypher projection, as its name implies, uses Cypher to define the projection pattern, and as such, enables more flexibility.
```
Comment (adam-cowley) [inline asciidoc/courses/gds-product-introduction/modules/2-graph-management/lessons/4-cypher-projection/lesson.adoc:13]:
```
While the native projection requires less configuration, its filtering and aggregation capabilities aren’t as flexible as Cypher. The Cypher projection, as its name implies, uses Cypher to define the projection pattern and enables more flexibility.
```

File: asciidoc/courses/gds-product-introduction/modules/2-graph-management/lessons/3-c-native-projection/lesson.adoc (hunk 1 above)
Comment (FlorentinD) [reply asciidoc/courses/gds-product-introduction/modules/2-graph-management/lessons/3-c-native-projection/lesson.adoc:16]: I wanted to express `ease of use` as used at https://neo4j.com/docs/graph-data-science/current/management-ops/graph-creation/. 

`simplest` is not any better is it?

Any other phrasing ideas?

**Comment by martinohanlon** [comment]
@adam-cowley @FlorentinD  Thanks for the comments. Ill make the changes suggested, If you look through the rest of the course you will find similar occurrences. I have plans for a complete overhaul of these courses so I am keeping the changes light to avoid rework.

File: asciidoc/courses/gds-product-introduction/modules/2-graph-management/lessons/3-c-native-projection/lesson.adoc (hunk 1 above)
Comment (martinohanlon) [reply asciidoc/courses/gds-product-introduction/modules/2-graph-management/lessons/3-c-native-projection/lesson.adoc:16]: Easiest can be demoralizing. What we consider "easy" someone might consider hard. I have changed to quickest.

**Pull request merged**